lib/seahorn/Bmc.cc

```diff
@@ -130,205 +129,11 @@ raw_ostream &BmcEngine::toSmtLib(raw_ostream &out) {
   return m_smt_solver.toSmtLib(out);
 }
 
-BmcTrace BmcEngine::getTrace() {
+template <> BmcTrace<BmcEngine, ZModel_ref> BmcEngine::getTrace() {
```

Comment:
this template specialization is unexpected here. Probably issue somewhere else in the design. 
I expect not to have `template<>` in front of this method.